drivers/soundwire/bus.c

```diff
@@ -1414,7 +1414,7 @@ static int sdw_handle_slave_alerts(struct sdw_slave *slave)
 	u8 clear = 0, bit, port_status[15] = {0};
 	int port_num, stat, ret, count = 0;
 	unsigned long port;
-	bool slave_notify = false;
+	bool slave_notify;
```

Comment:
that's why you want to scope variables properly: If this variable was declared within the scope, where it's needed, i.e. inside the loop, you wouldn't have this issue ;-)

Reply:
we're past that point really... this code is what it is.